### DISCUSSION TITLE
Recycling containers: General waste

### DISCUSSION THREAD

**Discussion by fxedel** [streetcomplete/StreetComplete#5736] [Ideas]
In Denmark, and possibly also lots of other countries, recycling container areas often also include a container for general / residual waste – which is technically not a recycling container, since it probably won't be recycled. When answering SC's quest "What can be left here for recycling?", I'm always wondering what to do about the general waste container. I don't know how to map it in OSM either. Some options would be:

* Set [recycling:waste=yes](https://wiki.openstreetmap.org/wiki/Key:recycling) – although the wiki says "don't use this if the waste is not recycled, use a tag like [amenity](https://wiki.openstreetmap.org/wiki/Key:amenity)=[waste_disposal](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dwaste_disposal) or [amenity](https://wiki.openstreetmap.org/wiki/Key:amenity)=[waste_basket](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dwaste_basket) instead"
* Map waste basket separately as suggested by wiki – which I find a bit ugly, because it's all containers of the same shape. If the containers are mapped as an area, should I place some bins within that area then? It also feels like micromapping, when I start adding nodes for different kinds of containers.
* Add [bin=yes](https://wiki.openstreetmap.org/wiki/Key%3Abin) or [waste=trash](https://wiki.openstreetmap.org/wiki/Key%3Awaste) – but [taginfo](https://taginfo.openstreetmap.org/tags/recycling_type=container#combinations) says that these one only very rarely used

What is your opinion on how to map these waste containers and would there be a possibility to adapt the SC quest such that it's more convenient to map them?

![IMG_20240708_210157](https://github.com/streetcomplete/StreetComplete/assets/7782229/51e45fe3-2f59-4bfb-b6ad-b23a7b7d871d)

**Comment by HolgerJeromin** [comment]
The netherlands has the same.
Glas and paper are open to the public. waste needs an rfid chip

**Comment by mnalis** [comment]
there has recently been a mess with that wiki, see [Talk page](https://wiki.openstreetmap.org/wiki/Talk:Tag:amenity%3Drecycling#recycling:waste_(and_possibly_others)_definition_change?) for details... That being said, two points:

- for general discussion on how something should be tagged, one should ask on [community tagging subforum](https://community.openstreetmap.org/c/general/tagging/70) or [Tagging mailing list](https://wiki.openstreetmap.org/wiki/Tagging_mailing_list).  When we have consensus there, we can document it on the wiki and then implement it in StreetComplete. (asking here is much less efficient, as StreetComplete GitHub discussions followers are just a tiny minority of OSM users...)
- I personally would map (via Things Overlay) separate node with [amenity=waste_disposal](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dwaste_disposal) at the approximate location inside that `amenity=recycling` polygon and call it a day. Other people might leave an OSM Note and leave it to others to handle... (If it is only garbage dumpster without any recyclables, there is `Uh...` answer for that already)

**Reply by matkoniecz** [reply]
General tagging decision really should not be taken on issue tracker of specific project.

**Reply by fxedel** [reply]
Okay, I might need to clarify: My main issue is that answering SC's quest "What can be left here for recycling?" is unsatisfactory if there is no easy option for general waste as well. This might be a minor issue in countries where it is uncommon to have these general containers together with other recycling containers, but it is very common for example in Denmark. I just pointed out that there might be different ways to tag these containers and I don't know which of these is most desirable; depending on that, it would be nice to extend the SC quest (at least in some countries) such that we can easily map these containers within SC.

**Reply by mnalis** [reply]
Sure @fxedel, and what you ask for is a **good idea** (and likely for more places than just Denmark)! :heart: 

Only problem is that it cannot be solved by StreetComplete, **_until_** there is first agreement among general OpenStreetMap community **which tags** should be used for such case -- and there currently seem to be _no consensus_ except mapping it as separate node with [amenity=waste_disposal](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dwaste_disposal) (which I agree with you is quite annoying workaround).

While you'll reach maybe a dozen or so mappers on StreetComplete GitHub discussions, that is too small a community to decide on tagging issues, thus my suggestion to bring your question to [community tagging subforum](https://community.openstreetmap.org/c/general/tagging/70). If you do, please do add link to that discussion here, so interested parties may follow! When OSM community decides (and documents on the wiki) how to tag such objects containing both recycling containers and non-recyclable general waste,  ***then***  one can open issue at StreetComplete GitHub to suggest using those new tags dedicated for that purpose.

**Comment by FloEdelmann** [comment]
See also this recently opened issue:

* #5753

**Comment by mnalis** [comment]
related: https://community.openstreetmap.org/t/is-amenity-waste-basket-recycling-good-for-tagging-combined-garbage-recycling-bins/101616
